pkg/volume/volume.go

```diff
@@ -165,7 +165,7 @@ type Attacher interface {
 // Detacher can detach a volume from a node.
 type Detacher interface {
 	// Detach the given device from the node with the given Name.
-	Detach(deviceName string, nodeName types.NodeName) error
+	Detach(deviceName string, nodeName types.NodeName, spec *Spec) error
```

Comment:
To match how the Attacher interface looks like, should the *Spec be the first argument?